pkgs/games/xu4/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, zlib
+, libpng
+, boron
+, libGL
+, libX11
+, libXcursor
+, faun
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xu4";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "xu4-engine";
+    repo = "u4";
+    rev = "v${version}";
+    hash = "sha256-3jyUSVfmjuKsHQKluFnQ8Gtg7aAI/WBjmY109BdNdyY=";
+    fetchSubmodules = true;
+  };
+
+  # this is not a standard Autotools-like `configure` script
+  dontAddPrefix = true;
+
+  preConfigure = ''
+    patchShebangs configure
+  '';
+
+  postConfigure = ''
+    # need to report this upstream
+    sed --in-place 's/-Wall/-Wno-error=format-security/g' src/Makefile
+
+    # disable git submodule init as part of build
+    sed --in-place 's/^\s*git submodule.*//g' src/Makefile.common
+
+    # set binary version (ref https://github.com/NixOS/nixpkgs/commit/571e02812f5da12fd851557ebbacea72fc7b5121#r129407907 )
+    sed --in-place 's/DR-1.0/${version}/' src/Makefile.common
+  '';
+
+  enableParallelBuilding = true;
+
+  buildInputs = [
+    zlib
+    libpng
+    boron
+    libGL
+    libX11
+    libXcursor
+    faun
+  ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  # The `install` target references some files with unknown license
+  installPhase =
```

Comment:
```suggestion
  installPhase = ''
```

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/181866/commits/467e54a9d3a81db5f064aa834b44eead13124b59